doc/ es/ user/ source/ services/ wms/configuration.rst

```diff
@@ -1,3 +1,4 @@
+@fabianfernandezcaatro47
```

Comment:
What is this change for?

Reply:
Este PR pasa los controles de control de calidad (QA comprueba los resultados serán reportados por travis-ci después de abrir este PR)

Reply:
The QA build passed regularly. This change broke the documentation build thought.